completions/ssh-keygen

```diff
@@ -80,10 +81,17 @@ _ssh_keygen()
                         compopt -o filenames
                         COMPREPLY=($(compgen -c -- "${cur#*=}"))
                         ;;
-                    checkpoint=* | challenge=*)
+                    checkpoint=* | challenge=* | write-attestation=*)
                         cur=${cur#*=}
                         _filedir
                         ;;
+                    application=)
+                        COMPREPLY=("ssh:")
```

Comment:
Currently, any string already input by the user other than `ssh:` is removed. Maybe this is more natural:
```suggestion
                        COMPREPLY=($(compgen -W "ssh:" -- "${cur#*=}"))
                        ((${#COMPREPLY[@]} == 0)) && COMPREPLY=("ssh:${cur#*=}")
```

Reply:
Thanks, added the first in b252360. Left out the second because it seems a bit too magical to me and would give incorrect results if new application prefixes are ever introduced.

Reply:
OK!